packages/clay-css/src/scss/components/_empty-state.scss

```diff
@@ -63,3 +63,132 @@
 		@include clay-css(map-get($c-empty-state-sm, c-empty-state-footer));
 	}
 }
+
+// Empty State Animation
+.c-empty-state-animation {
+	.empty-state-animation {
+		@media (prefers-reduced-motion: no-preference) {
+			.ship {
+				transform-origin: bottom;
+				animation: 6s ease-in-out infinite ship;
+			}
```

Comment:
We need to add new lines after each rule set. Idk why `yarn format` doesn't do this.